galaxy_ng/app/settings.py

```diff
@@ -65,6 +65,16 @@
 # set to 'insights' for cloud.redhat.com deployments
 GALAXY_DEPLOYMENT_MODE = 'standalone'
 
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        "rest_framework.authentication.SessionAuthentication",
+        "rest_framework.authentication.TokenAuthentication",
+        "rest_framework.authentication.BasicAuthentication",
+        "dynaconf_merge",
```

Comment:
I think this `dynaconf_merge` inside the list is not needed, you already have `dynaconf_merge: True` at the root of the dictionary.

I just tested


```py
>>> from dynaconf import Dynaconf
>>> settings = Dynaconf(REST_FRAMEWORK={"DEFAULT_AUTHENTICATION_CLASSES": ("a", "b", "c")})
>>> settings.REST_FRAMEWORK
<Box: {'DEFAULT_AUTHENTICATION_CLASSES': ['a', 'b', 'c']}>

>>> settings.set("REST_FRAMEWORK", {"DEFAULT_AUTHENTICATION_CLASSES": ("d","e"), "dynaconf_merge": True})
>>> settings.REST_FRAMEWORK
<Box: {'DEFAULT_AUTHENTICATION_CLASSES': ['a', 'b', 'c', 'd', 'e']}>
```